### ISSUE TITLE
layout are not problem

### ISSUE THREAD

**Issue by letarg0** [shapr/markovkeyboard#4]
Problem are with keys,
i have 32 chars in my alphabet, french have more. I still need programming chars and ... 
this is a trouble. 

In my opinion change this layout for more than 105 keys

**Comment by shapr** [comment]
I agree! My "native" keyboard layout is dvorak instead of qwerty, and this doesn't work so well for me!
I'm not sure how to integrate this for a French layout or other non-English, do you have any suggestions?

**Comment by xor256** [comment]
Look this https://klawiatura.wordpress.com/
